### PR TITLE
[ONNX] Add export of aten::is_floating point

### PR DESCRIPTION
**Pull request by KsenijaS**
Add export of aten::is_floating point

### CODE REVIEW

File: scripts/onnx/test.sh [hunk 1]
```diff
@@ -51,6 +51,7 @@ pytest "${args[@]}" \
   --ignore "$top_dir/test/onnx/test_custom_ops.py" \
   --ignore "$top_dir/test/onnx/test_models_onnxruntime.py" \
   --ignore "$top_dir/test/onnx/test_utility_funs.py" \
+  --ignore "$top_dir/test/onnx/test_pytorch_onnx_shape_inference.py" \
```
Comment (BowenBao) [inline scripts/onnx/test.sh:54]: Just leaving a note here, this is to exclude a test for now that is WIP.

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 2]
```diff
@@ -849,6 +849,58 @@ def test_avgpool_3d_ceil(self):
...
+                    return x.new_zeros(x.shape)
+                return x.new_zeros(x.shape)
+
+        x = torch.randn(2, 3, 4)
+        self.run_test(FloatingPoint(), x)
+
+        class FloatingPoint(torch.jit.ScriptModule):
+            @torch.jit.script_method
+            def forward(self, x):
+                if x.size(0) > 1:
```
Comment (BowenBao) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:866]: why need this condition in the test case, is it affecting the the test case somehow?

Comment (KsenijaS) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:866]: I needed to create a case where a type could not be inferred from torch traced graph and nested if statement represent one of those cases.

**Review by neginraoof — APPROVED** [review]
Looks good. Thanks!

**Review by BowenBao — APPROVED** [review]

**Comment by KsenijaS** [comment]
@bzinodev can you please review? Thanks!

**Review by bzinodev — APPROVED** [review]

**Pull request closed without merging**